Sources/Frame-iOS/Networking/CustomerIdentities/CustomerIdentityEndpoints.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  File.swift
+//  Frame-iOS
+//
+//  Created by Frame Payments on 8/12/25.
+//
+
+import Foundation
+
+enum CustomerIdentityEndpoints: FrameNetworkingEndpoints {
+    //MARK: Customer Identity Endpoints
+    case createCustomerIdentity
+    case getCustomerIdentityWith(customerIdentityId: String)
+    
+    var endpointURL: String {
+        switch self {
+        case .createCustomerIdentity:
+            return "/v1/customer_identity_verifications"
+        case .getCustomerIdentityWith(let customerIdentityId):
+            return "/v1/customer_identity_verifications\(customerIdentityId)"
```

Comment:
I think `/` is missing here in the path

```suggestion
            return "/v1/customer_identity_verifications/\(customerIdentityId)"
```